templates/landscape/index.html

```diff
@@ -3,6 +3,7 @@
 {% block title %}Landscape{% endblock %}
 {% block meta_description %}Landscape is the leading management and administration tool for all versions of Ubuntu, anywhere.{% endblock meta_description %}
 {% block meta_copydoc %}https://docs.google.com/document/d/1Q5KPVOn3yaexgSbkZHmXHmWzT5WG3ocYAPthweMwDz4/edit{% endblock meta_copydoc %}
+{% block meta_image %}https://assets.ubuntu.com/v1/742fa591-Canonical-Landscape-logo-2022.png{% endblock meta_image %}
```

Comment:
This format of meta image is not supported
You can test here: https://www.heymeta.com/url/ubuntu-com-12622.demos.haus/landscape

![image](https://user-images.githubusercontent.com/2707508/222745345-34c52c8f-4cb0-412d-89f3-b34fc14b3d48.png)

Reply:
What do you mean the format of the image? .png is not supported? or is it the size of the image?

We may need to ask a designer then to create one custom for this.

Reply:
Checking here I don't see any issues, but the image size does get cropped https://www.opengraph.xyz/url/https%3A%2F%2Fubuntu-com-12622.demos.haus%2Flandscape

Reply:
@carkod just got an asset for you! https://assets.ubuntu.com/v1/6b9a0b06-landscape-metatag.svg

Reply:
Updated. Thanks.